physics/CONV/Chikira_Sugiyama/cs_conv.F90

```diff
@@ -3940,8 +4184,8 @@ SUBROUTINE CUMCHK                                   & ! check range of output va
       REAL(kind_phys) :: CLWMAX  = 1.e-3_kind_phys
       REAL(kind_phys) :: TPRPMAX = 1.e-2_kind_phys
       REAL(kind_phys) :: GTQIMAX = 1.e-5_kind_phys
-      !REAL(kind_phys) :: GTM2MAX = 1._kind_phys
-      !REAL(kind_phys) :: GTM3MAX = 1._kind_phys
+      REAL(kind_phys) :: GTM2MAX = 1._kind_phys
+      REAL(kind_phys) :: GTM3MAX = 1._kind_phys
```

Comment:
GTM2MAX and GTM3MAX are not used in subroutine CUMCHK, and can be removed or comment out